src/utils/googleTagManager.ts

```diff
@@ -129,18 +152,20 @@ export const trackEvent = ({
   event: GTM_EVENT
   category: string
   action: string
-  label?: string | number | boolean
+  label?: EventLabel
 }): void => {
   const dataLayer = {
     event,
     chainId: _getChainId(),
     eventCategory: category,
     eventAction: action,
-    eventLabel: label,
+    eventLabel: tryParse(label),
+    // Flag used to prevent GA triggers
+    dev: !IS_PRODUCTION,
```

Comment:
Can't we use the env-1, env-2 etc for that?

Reply:
Very true. The development environment has an in-built debug mode variable. I've adjusted the trigger to only fire when it's `false` (in the production environment).